test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_ListDescribeConsumerGroups.cs

```diff
@@ -63,16 +63,68 @@ private void checkConsumerGroupDescription(
         [Theory, MemberData(nameof(KafkaParameters))]
         public void AdminClient_ListDescribeConsumerGroups(string bootstrapServers)
         {
+            var groupID = Guid.NewGuid().ToString();
+            var nonExistentGroupID = Guid.NewGuid().ToString();
             if (!TestConsumerGroupProtocol.IsClassic())
             {
-                LogToFile("KIP 848 Admin operations changes still aren't " +
```

Comment:
Why was this removed?

Reply:
Because these will be operational once the PR goes in and even in preview mode if unsafe api kafka server setting is set, then these are operational.